WordPress/gutenberg
# Fix: Ensure consistency in editor tools for navigation buttons and delete options

Fixes: #63991


## What?
This PR addresses inconsistencies in the editor tools UI, focusing on:

Navigation Button Design: Ensures consistent use of chevrons for menu items like Shadows and Typography/Font Sizes, which navigate to edit screens.
Delete All Options: Adds or aligns "Delete All" functionality for custom settings like Shadows, Custom Font Sizes, and Colors.

## Why?
Inconsistent UI elements create confusion for users and hinder the overall experience. Aligning the design and functionality of editor tools improves usability, accessibility, and coherence across the interface.

## How?
Updated navigation button components to uniformly display chevrons for items that navigate to sub-settings (e.g., Shadows, Typography/Font Sizes).
Implemented consistent "Delete All" options for applicable settings, ensuring uniform behavior across Shadows, Custom Font Sizes, and Colors.
Adjusted CSS and component logic for styling and interactions to reflect these changes.

## Testing Instructions

1. Navigation Buttons

    - Check that Shadows and Typography/Font Sizes now show chevrons consistently.

2. Delete All Options

    - Verify the "Delete All" option exists and works for Shadows, Custom Font Sizes, and Colors.

## Screenshots or screencast 

https://github.com/user-attachments/assets/dfb019df-d454-4ba4-abb1-7d0456af6d85
